foxpuppet/api/windows.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from contextlib import contextmanager
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+from foxpuppet.foxpuppet import FoxPuppet
+
+
+class Windows(FoxPuppet):
+
+    def __init__(self, selenium):
```

Comment:
Only use selenium here when you have to use it. Otherwise use *args and **kwargs and pass everything through to the super class.